assets/source/setup-guide/app/steps/BillingStatus.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @wordpress/no-global-event-listener */
+/**
+ * External dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+import { Card } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import Billing from '../components/Billing/Billing';
+import StepOverview from '../components/StepOverview';
+import { useSettingsSelect } from '../helpers/effects';
+
+const BillingStatus = () => {
+	const appSettings = useSettingsSelect();
```

Comment:
Since `appSettings` is not really used within this `BillingStatus` component but is being passed down into `Billing` child component, we can just move this line into `Billing`. So `Billing` does not need to have the `accountData` prop.